src/test/coverartutils_test.cpp

```diff
@@ -73,23 +69,23 @@ TEST_F(CoverArtUtilTest, extractEmbeddedCover) {
                 kTestDir.absoluteFilePath("cover-test-jpg.mp3"), referenceJPGImage);
     }
 
-    if (isSupportedFileExtension("ogg")) {
+    if (SoundSourceProxy::isFileSuffixSupported(QStringLiteral("ogg"))) {
         extractEmbeddedCover(
                 kTestDir.absoluteFilePath("cover-test.ogg"), referencePNGImage);
     }
 
-    if (isSupportedFileExtension("opus")) {
+    if (SoundSourceProxy::isFileSuffixSupported(QStringLiteral("opus"))) {
         // opus
         extractEmbeddedCover(
                 kTestDir.absoluteFilePath("cover-test.opus"), referencePNGImage);
     }
 
-    if (isSupportedFileExtension("wav")) {
+    if (SoundSourceProxy::isFileSuffixSupported(QStringLiteral("wav"))) {
         extractEmbeddedCover(
                 kTestDir.absoluteFilePath("cover-test.wav"), referencePNGImage);
     }
 
-    if (isSupportedFileExtension("wv")) {
+    if (SoundSourceProxy::isFileSuffixSupported(QStringLiteral("wv"))) {
         extractEmbeddedCover(
                 kTestDir.absoluteFilePath("cover-test.wv"), referencePNGImage);
     }
```

Comment:
mod and ogg are critical and deserves a test IMHO

Reply:
Unrelated. I didn't change this code and won't extend the scope of this PR.

Reply:
You change the code around file extension handling here and @Holzhaus in https://github.com/mixxxdj/mixxx/pull/4602 At one point it is IMHO mandatory that we have a test for these cases I consider as critical. 

Isn't it just a copy and paste task here?

Reply:
Just checked: ogg is already tested, mod does not support cover art. What are you asking for here?

Reply:
Ah Ok I did not know this. But it looks like we have also nowhere else a test with ```SoundSourceProxy::isFileSuffixSupported(QStringLiteral("mod"))``` 
Or "mod" in general l. 

Maybe we can put it here anyway with no cover expected? 

But the original issue we need to test if the magicNumber -> mimeType -> preeredSuffix -> fileType -> suffix round trip still works.  
Not sure if this is already covered by a test or if we need to check it explicit. 

I will explain the details in the other thread below.

Reply:
https://github.com/mixxxdj/mixxx/pull/4608/commits/e40fab905e5d3a84e3f9c63bfbda924f90e0545f